.github/workflows/test.yaml

```diff
@@ -89,7 +93,7 @@ jobs:
           nix develop .#ci --command bash ./scripts/wait-for-hc.sh
 
           # Run the scenario
-          RUST_LOG=info nix run .#zome_call_single_value -- --connection-string ws://localhost:8888 --duration 5 --no-progress
+          RUST_LOG=info cargo run --package zome_call_single_value -- --connection-string ws://localhost:8888 --duration 5 --no-progress
```

Comment:
As people had added to this workflow, a mixture of `cargo` and `nix` was being used. That meant we were building all the project dependencies twice and some of the test zomes twice. Just switched to cargo since we're running the tests with cargo anyway